runtime/sqlite_record_logger.hpp

```diff
@@ -56,14 +56,18 @@ class sqlite_thread {
 				create_table_string += std::string{"INTEGER"};
 			} else if (rh.get_column_type(i) == typeid(bool)) {
 				create_table_string += std::string{"INTEGER"};
+			} else if (rh.get_column_type(i) == typeid(double)) {
+				create_table_string += std::string{"REAL"};
 			} else if (rh.get_column_type(i) == typeid(std::chrono::nanoseconds)) {
 				create_table_string += std::string{"INTEGER"};
 			} else if (rh.get_column_type(i) == typeid(std::chrono::high_resolution_clock::time_point)) {
```

Comment:
It is for performance timing.

Reply:
Oh, right. Should performance timing timestamps also move to `RelativeClock` in the future?

Reply:
No, they should move to using scope timer (#208).

Reply:
In the long run, we should be using a [new timing infrastructure][1]. In the mean time, let's leave this unchanged with the understanding that it will be deleted when we implement #208. (Maybe throw a comment saying as much in there)

[1]: https://github.com/charmoniumQ/scope_timer/